packages/@ourworldindata/explorer/src/ExplorerProgram.ts

```diff
@@ -366,30 +396,40 @@ export class ExplorerProgram extends GridProgram {
         return clone
     }
 
-    constructExplorerGrapherConfig(grapherRow: any): ExplorerGrapherInterface {
+    /**
+     * Grapher config for the currently selected row including global settings.
+     * Includes all columns that are part of the GrapherGrammar.
+     */
+    get explorerGrapherConfig(): ExplorerGrapherInterface {
+        const selectedGrapherRow = this.decisionMatrix.selectedRow
+
         const rootObject = trimAndParseObject(this.tuplesObject, GrapherGrammar)
         let config = { ...rootObject }
 
-        if (grapherRow && Object.keys(grapherRow).length) {
+        if (selectedGrapherRow && Object.keys(selectedGrapherRow).length) {
             config = {
                 ...config,
-                ...trimAndParseObject(grapherRow, GrapherGrammar),
+                ...trimAndParseObject(selectedGrapherRow, GrapherGrammar),
             }
         }
 
-        // remove all keys that are not part of the GrapherGrammar
+        // Remove all keys that are not part of the GrapherGrammar
         Object.keys(config).forEach((key) => {
             if (!GrapherGrammar[key]) delete config[key]
         })
 
         return config
     }
 
-    constructGrapherConfig(
-        explorerGrapherConfig: ExplorerGrapherInterface
-    ): GrapherInterface {
+    /**
+     * Grapher config for the currently selected row, with explorer-specific
+     * fields translated to valid GrapherInterface fields.
+     *
+     * For example, `yAxisMin` is translated to `{yAxis: {min: ... }}`
+     */
+    get grapherConfig(): GrapherInterface {
```

Comment:
Same here. No real change. Just got rid of the `constructGrapherConfig` indirection